pallets/block-rewards/src/lib.rs

```diff
@@ -314,10 +315,13 @@ impl<T: Config> Pallet<T> {
 	///  * deposit_stake (4 reads, 4 writes): Currency, Group, StakeAccount,
 	///    Account
 	pub(crate) fn do_init_collator(who: &T::AccountId) -> DispatchResult {
+		let existential_deposit =
+			<T::Tokens as Inspect<T::AccountId>>::minimum_balance(T::StakeCurrencyId::get());
+
 		<T::Tokens as Mutate<T::AccountId>>::mint_into(
 			T::StakeCurrencyId::get(),
 			who,
-			T::StakeAmount::get().saturating_add(T::ExistentialDeposit::get()),
```

Comment:
Here, we were using the CFG existential deposit instead of the one from `StakeCurrencyId`